aggregation/src/main/java/fk/prof/aggregation/serialize/Serializer.java

```diff
@@ -22,6 +23,16 @@ static void writeFixedWidthInt32(int value, OutputStream os) throws IOException
         os.write(bytes);
     }
 
+    static void writeVariantInt32(int value, OutputStream os) throws IOException {
```

Comment:
Add a TODO somewhere relevant that we are going to switch to varint32 everywhere in aggregation proto (and do away with fixedwidth uint32 serialization)